Framework/DataObjects/src/FractionalRebinning.cpp

```diff
@@ -20,7 +20,8 @@ namespace FractionalRebinning {
 
 /**
  * Find the possible region of intersection on the output workspace for the
- * given polygon. The given polygon must have a CLOCKWISE winding.
+ * given polygon. The given polygon must have a CLOCKWISE winding and the
```

Comment:
Thanks for recording this. Hopefully it will make other peoples lives easier now.